models/falcon-40b-instruct.json

```diff
@@ -0,0 +1,86 @@
+{
+  "_descriptorVersion": "0.0.1",  
+  "datePublished": "2023-10-22T03:04:42",
+  "name": "falcon-40b-instruct",
+  "description": "Falcon-40B-Instruct, based on Falcon-40B, has been fine-tuned on chat and instruct datasets. It offers outstanding performance and is an excellent choice for chat and instruct applications. Falcon-7B-Instruct is part of the Falcon family of language models, known for their exceptional capabilities and openness.",
+  "author": {
```

Comment:
The `author` field is intended for the model creator, so in this case it would talk about tiiuae. See other files for example